src/cljs/main/broadfcui/page/method_repo/method/details.cljs

```diff
@@ -223,8 +223,9 @@
         {:endpoint (endpoints/get-agora-method namespace name snapshot-id)
          :on-done (net/handle-ajax-response
                    (fn [{:keys [success? parsed-response]}]
+                     (swap! state assoc :loading-snapshot? false)
                      (if success?
-                       (swap! state assoc :selected-snapshot parsed-response :loading-snapshot? false)
+                       (swap! state assoc :selected-snapshot parsed-response)
```

Comment:
in the error case - i.e. `(not success?)` - we never reset `:loading-snapshot?` to false. Therefore, the spinner at [line 51](https://github.com/broadinstitute/firecloud-ui/blob/develop/src/cljs/main/broadfcui/page/method_repo/method/details.cljs#L50) of this same file never disappeared.